src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -287,8 +318,23 @@ void UserInfoPopup::updateUserData()
             }
         }
 
+        // get ignoreHighlights state
+        bool isIgnoringHighlights = false;
+        auto ignoreHighlightsUsersVector = getApp()->highlights->blacklistedUsers.getVector();

```

Comment:
Use `const auto  &` so the vector doesn't get copied.

Rename the vector name to `vector` (same reason as above).